src/api/tmc.ts

```diff
@@ -275,7 +307,8 @@ export default class TMC {
             if (response.status === 403) {
                 return new Err(new AuthorizationError("403 - Forbidden"));
             }
-            return new Err(new ApiError(response.status + " - " + response.statusText));
+            const errorText = await response.text();
+            return new Err(new ApiError(`${response.status} - ${response.statusText} - ${errorText}`));
```

Comment:
Changed to use a template string here; it may have been possible to inline `errorText` like it was in the suggestion, but this shouldn't hurt either.